bls.go

```diff
@@ -365,6 +366,12 @@ func (a *blsBLS12381Algo) decodePublicKeyCompressed(publicKeyBytes []byte) (Publ
 	return a.decodePublicKey(publicKeyBytes)
 }
 
+// signatureFormatCheck verifies the format of a serialized signature,
+// regardless of messages or public keys.
+func (a *blsBLS12381Algo) signatureFormatCheck(sig Signature) bool {
+	return len(sig) == SignatureLenBLSBLS12381
```

Comment:
The function should panic instead of returing a reault. The panic message is  `"%s does not support signature format check, a.algo"`